### PR TITLE
docs: move and update localization documentation

### PR DESCRIPTION
**Pull request by fbergkemper**
#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

### Checklist for the _reviewer_ of the PR (will be processed by the Bareos team)
Make sure you check/merge the PR using `devtools/pr-tool` to have some simple automated checks run and a proper changelog record added.

##### General
- [x] Is the PR title usable as CHANGELOG entry?
- [x] Purpose of the PR is understood
- [x] Commit descriptions are understandable and well formatted
- [x] Check backport line

##### Source code quality
- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR

### CODE REVIEW

File: docs/manuals/source/DeveloperGuide/Webui.rst [hunk 1]
```diff
@@ -4,6 +4,46 @@
...
 
+.. _section-dev-webui-translation:
+
+Translation
+-----------
+
+Where do I find the translation?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The localization files are located in `module/Application/language`. You can find the languages that are currently
```
Comment (joergsteffens) [inline docs/manuals/source/DeveloperGuide/Webui.rst:15]:
```suggestion
The localization files are located in `webui/module/Application/language`. You can find the languages that are currently
```

File: docs/manuals/source/DeveloperGuide/Webui.rst [hunk 2]
```diff
@@ -4,6 +4,46 @@
...
+in translation in our online project, which is addressed in the next section.
+
+How should I edit translation files?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You need to join our online project for translating the Bareos WebUI. We use `http://poeditor.com`_.
```
Comment (joergsteffens) [inline docs/manuals/source/DeveloperGuide/Webui.rst:21]:
```suggestion
You need to join our online project for translating the Bareos WebUI. We use https://poeditor.com.
```

File: docs/manuals/source/DeveloperGuide/Webui.rst [hunk 3]
```diff
@@ -4,6 +4,46 @@
...
+The following link, `https://poeditor.com/join/project/ELnLNbvQJb`_ , will redirect you to the right project.
+You can join our translation community as a contributor there.
+
+How can I add a new language?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We will add the feature to add a new lanugage as a contributor after the existing localizations have been completed.
+Check back in a while to see if it's possible then.
```
Comment (joergsteffens) [inline docs/manuals/source/DeveloperGuide/Webui.rst:29]:
```suggestion
We contact us, if you want to contribute a new language. We will prepare https://poeditor.com for that.
```

File: docs/manuals/source/DeveloperGuide/Webui.rst [hunk 4]
```diff
@@ -4,6 +4,46 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the script `webui/scripts/localization_update.sh` to scan for changes and generate new \*.po(t) files.
+The Bareos Team takes care of this task and the synchronization with POEditor.
+
+What if my questions are not answered in this document?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Do not be afraid to ask them on our mailinglist, bareos-devel@googlegroups.com .
+
```
Comment (joergsteffens) [inline docs/manuals/source/DeveloperGuide/Webui.rst:46]:
```suggestion
```

**Review by joergsteffens — CHANGES_REQUESTED** [review]

File: webui/doc/README-INSTALLATION.md [hunk 5]
```diff
@@ -1,3 +1,5 @@
+
 INSTALLATION
 ============
+
 Please see: https://docs.bareos.org/IntroductionAndTutorial/InstallingBareosWebui.html
```
Comment (bruno-at-bareos) [inline webui/doc/README-INSTALLATION.md:5]: Maybe better to use the :ref:`section-webui` so it will point to the right page (actually BareosWebui.html)

Comment (arogge) [reply webui/doc/README-INSTALLATION.md:5]: that won't work. It's markdown not ReST.

**Review by joergsteffens — APPROVED** [review]

**Pull request merged**